demisto_sdk/commands/prepare_content/integration_script_unifier.py

```diff
@@ -393,6 +393,22 @@ def get_script_or_integration_package_data(package_path: Path):
 
         return yml_path, code
 
+    @staticmethod
+    def get_script_or_integration_package_data_with_sha(
```

Comment:
can't we combine it with the other `get_script_or_integration_package_data` that gets an arugment of git_sha? and if the git_sha is none will get it from local path, otherwise from sha.

Reply:
They have different logic and different args, I think it's better keeping it this way.